ui/pages/settings/settings-tab/settings-tab.component.js

```diff
@@ -136,21 +143,85 @@ export default class SettingsTab extends PureComponent {
 
   renderBlockieOptIn() {
     const { t } = this.context;
-    const { useBlockie, setUseBlockie } = this.props;
+    const {
+      useBlockie,
+      setUseBlockie,
+      selectedAddress,
+      useTokenDetection,
+      tokenList,
+    } = this.props;
+
+    const getStyles = (diameter) => ({
+      height: diameter,
+      width: diameter,
+      borderRadius: diameter,
+      border: '2px solid #037DD6',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    });
+
+    const getIconStyles = () => ({
+      display: 'block',
+      borderRadius: '16px',
+      width: '32px',
+      height: '32px',
+    });
 
     return (
       <div className="settings-page__content-row" id="blockie-optin">
         <div className="settings-page__content-item">
-          <span>{this.context.t('blockiesIdenticon')}</span>
-        </div>
-        <div className="settings-page__content-item">
-          <div className="settings-page__content-item-col">
-            <ToggleButton
-              value={useBlockie}
-              onToggle={(value) => setUseBlockie(!value)}
-              offLabel={t('off')}
-              onLabel={t('on')}
-            />
+          <Typography variant={TYPOGRAPHY.H5} color={COLORS.BLACK}>
+            {t('accountIdenticon')}
+          </Typography>
+          <span className="settings-page__content-item__description">
+            {t('jazzAndBlockies')}
+          </span>
+          <div className="settings-page__content-item__identicon">
+            <div className="settings-page__content-item__identicon__item">
+              <div
+                data-test-id="jazz_icon"
+                onClick={() => setUseBlockie(false)}
+                style={useBlockie ? null : getStyles(40)}
```

Comment:
[REQ] this should be achieved with classes instead of inline styles.

Reply:
@brad-decker updated.